src/script/components/MessagesList/Message/index.tsx

```diff
@@ -95,11 +93,9 @@ const Message: React.FC<
     lastReadTimestamp,
     onVisible,
     scrollTo,
-    totalMessage,
```

Comment:
Those infos are not needed anymore to compute the focus index, only the messageId is necessary